agent/proto/agent_drpc_old.go

```diff
@@ -51,7 +51,17 @@ type DRPCAgentClient24 interface {
 	ReportConnection(ctx context.Context, in *ReportConnectionRequest) (*emptypb.Empty, error)
 }
 
-// DRPCAgentClient25 is the Agent API at v2.5.
+// DRPCAgentClient25 is the Agent API at v2.5. It adds a ParentId field to the
+// agent manifest response. Compatible with Coder v2.23+
 type DRPCAgentClient25 interface {
 	DRPCAgentClient24
 }
+
```

Comment:
It does seem odd to have an API version 2.5 that is equivalent to 2.4 with no changes. AFAICT to ship these changes with version 2.5 we would need to remove agent api version 2.5 from the current release branch. I'm not sure it's worth the hassle?

Reply:
Ah, then please disregard!